oneBarangay/static/assets/js/volt.js

```diff
@@ -15,166 +15,166 @@
+'use strict'
+const d = document
+d.addEventListener('DOMContentLoaded', function (event) {
   const swalWithBootstrapButtons = Swal.mixin({
     customClass: {
-      confirmButton: "btn btn-primary me-3",
-      cancelButton: "btn btn-gray",
+      confirmButton: 'btn btn-primary me-3',
+      cancelButton: 'btn btn-gray'
     },
-    buttonsStyling: false,
-  });
+    buttonsStyling: false
+  })
 
-  const themeSettingsEl = document.getElementById("theme-settings");
-  const themeSettingsExpandEl = document.getElementById("theme-settings-expand");
+  const themeSettingsEl = document.getElementById('theme-settings')
+  const themeSettingsExpandEl = document.getElementById('theme-settings-expand')
 
   if (themeSettingsEl) {
     const themeSettingsCollapse = new bootstrap.Collapse(themeSettingsEl, {
       show: true,
-      toggle: false,
-    });
+      toggle: false
+    })
 
-    if (window.localStorage.getItem("settings_expanded") === "true") {
-      themeSettingsCollapse.show();
-      themeSettingsExpandEl.classList.remove("show");
+    if (window.localStorage.getItem('settings_expanded') === 'true') {
+      themeSettingsCollapse.show()
+      themeSettingsExpandEl.classList.remove('show')
     } else {
-      themeSettingsCollapse.hide();
-      themeSettingsExpandEl.classList.add("show");
+      themeSettingsCollapse.hide()
+      themeSettingsExpandEl.classList.add('show')
     }
 
-    themeSettingsEl.addEventListener("hidden.bs.collapse", function () {
-      themeSettingsExpandEl.classList.add("show");
-      window.localStorage.setItem("settings_expanded", false);
-    });
+    themeSettingsEl.addEventListener('hidden.bs.collapse', function () {
+      themeSettingsExpandEl.classList.add('show')
+      window.localStorage.setItem('settings_expanded', false)
+    })
 
-    themeSettingsExpandEl.addEventListener("click", function () {
-      themeSettingsExpandEl.classList.remove("show");
-      window.localStorage.setItem("settings_expanded", true);
+    themeSettingsExpandEl.addEventListener('click', function () {
+      themeSettingsExpandEl.classList.remove('show')
+      window.localStorage.setItem('settings_expanded', true)
       setTimeout(function () {
-        themeSettingsCollapse.show();
-      }, 300);
-    });
+        themeSettingsCollapse.show()
+      }, 300)
+    })
   }
 
   // options
   const breakpoints = {
     sm: 540,
     md: 720,
     lg: 960,
-    xl: 1140,
-  };
+    xl: 1140
+  }
 
-  const sidebar = document.getElementById("sidebarMenu");
+  const sidebar = document.getElementById('sidebarMenu')
   if (sidebar && d.body.clientWidth < breakpoints.lg) {
-    sidebar.addEventListener("shown.bs.collapse", function () {
-      document.querySelector("body").style.position = "fixed";
-    });
-    sidebar.addEventListener("hidden.bs.collapse", function () {
-      document.querySelector("body").style.position = "relative";
-    });
+    sidebar.addEventListener('shown.bs.collapse', function () {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/PrynsTag/oneBarangay/pullRequest?prid=7999497)
```suggestion
        sidebar.addEventListener("shown.bs.collapse", function () {
```